server/server.py

```diff
@@ -74,11 +74,15 @@ def wrapper(self, *args, **kwargs):
             except InvalidTokenException:
                 return {"error": "invalid token"}, 400
             except UnauthorizedTokenException:
-                return {"error": "unauthorized token"}, 401
+                return {"error": "Unauthorized token"}, 401
             except BWSAPIRateLimitExceededException:
-                return {"error": "rate limited"}, 429
+                return {"error": "Rate limited"}, 429
```

Comment:
You've rewritten the 401 & 429 responses to be sentence case, but both of the 400 responses are still lower case. I don't mind which case is used, but they should all be consistent IMO.